s64da_benchmark_toolkit/prepare.py

```diff
@@ -4,11 +4,13 @@
 import shutil
 
 from collections import namedtuple
+from sys import exit
 
 from concurrent.futures import ThreadPoolExecutor, as_completed
 from psycopg2 import ProgrammingError
 from subprocess import Popen, PIPE
 from urllib.parse import urlparse
+import threading
```

Comment:
`threading` right after `shutil`, please (it is a bundled Python module and this would then be the recommended import order).